tests/RobotFramework/tests/pkcs11/tedge_cert_create/initial.robot

```diff
@@ -0,0 +1,13 @@
+*** Settings ***
+Resource        tedge_cert_create.resource
+
+Suite Setup     tedge-p11-server Setup    ${TEDGE_P11_SERVER_VERSION}
+
+
+*** Variables ***
+${TEDGE_P11_SERVER_VERSION}     ${EMPTY}
```

Comment:
We will have to update that line with the correct version once the PR actually merged.